app/views/service_providers/_form.html.erb

```diff
@@ -354,28 +354,32 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
-
-      <br><legend><b>Forgot password help text:</b></legend>
-      <%= h.input  "forgot_password[en]", label: "English", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "en") } %>
-      <%= h.input "forgot_password[es]", label: "Español", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "es") } %>
-      <%= h.input "forgot_password[fr]", label: "Français", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "fr") } %>
-    <% end %>
-  </fieldset>
-</fieldset>
+  <% if current_user.admin? %>
+    <fieldset class="usa-fieldset-inputs custom-help-text">
+      <legend>Custom help text</legend>
+      <p>You can specify help text in these different areas to alert users of important information for logging into your app. Permitted HTML elements include the following:
+      <%= ServiceProvider::ALLOWED_HELP_TEXT_HTML_TAGS.map { |t| "<code>#{t}</code>" }.join(", ").html_safe %>.
+      </p>
+      <%= form.fields_for :help_text do |h| %>
+        <br><legend><b>Sign-in help text:</b></legend>
+        <%= h.input "sign_in[en]", label: "English", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_in", "en") } %>
+        <%= h.input "sign_in[es]", label: "Español", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_in", "es") } %>
+        <%= h.input "sign_in[fr]", label: "Français", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_in", "fr") } %>
+
+        <br><legend><b>Sign-up help text:</b></legend>
+        <%= h.input "sign_up[en]", label: "English", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_up", "en") } %>
+        <%= h.input "sign_up[es]", label: "Español", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_up", "es") } %>
+        <%= h.input "sign_up[fr]", label: "Français", as: :text, required: false, input_html: { value: service_provider.help_text.dig("sign_up", "fr") } %>
+
+        <br><legend><b>Forgot password help text:</b></legend>
+        <%= h.input  "forgot_password[en]", label: "English", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "en") } %>
+        <%= h.input "forgot_password[es]", label: "Español", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "es") } %>
+        <%= h.input "forgot_password[fr]", label: "Français", as: :text, required: false, input_html: { value: service_provider.help_text.dig("forgot_password", "fr") } %>
+      <% end %>
+    </fieldset>
+  <% else %>
+    <p class="usa-label">Help text</p>
+    <p>Do you need to edit the custom help text for your application? <a href="https://logingov.zendesk.com/agent">Contact us</a>.</p>
```

Comment:
Should we be linking to `/agent` here? Shouldn't this just be `http://logingov.zendesk.com/`?

Reply:
that was the specified link in the ticket.. I just tested it out by signing out of zendesk and then pasting in the link with /agent and it seemed to work

Reply:
and by "seemed to work" I mean it sent me to the support dashboard for users rather than our partner dashboard

Reply:
Using a different account from my GSA email Login.gov account puts me into a sign-in loop when trying to get to /agent.

Reply:
oh weird.. how did it get there then? in that case, my apologies